nixos/modules/services/audio/mpd.nix

```diff
@@ -62,6 +60,14 @@ in {
         '';
       };
 
+      playlistDirectory = mkOption {
+        type = types.path;
```

Comment:
I think `path` is okay, it's just a string that begins with a slash: https://github.com/NixOS/nixpkgs-channels/blob/nixos-unstable/lib/types.nix#L167 It's a little unfortunate that the name is overloaded (I too tend to think of store paths in this context).